codbex-payment-invoice-ext/generate/SalesInvoicePayment/api/GenerateSalesInvoicePaymentService.ts

```diff
@@ -0,0 +1,70 @@
+import { SalesInvoiceRepository as SalesInvoiceDao } from "../../../../codbex-invoices/gen/codbex-invoices/dao/salesinvoice/SalesInvoiceRepository";
+import { CustomerPaymentRepository as CustomerPaymentDao } from "../../../../codbex-payments/gen/codbex-payments/dao/CustomerPayment/CustomerPaymentRepository";
+import { SalesInvoicePaymentRepository as SalesInvoicePaymentDao } from "../../../../codbex-invoices/gen/codbex-invoices/dao/salesinvoice/SalesInvoicePaymentRepository";
+
+import { Controller, Get } from "sdk/http";
+
+@Controller
+class GenerateSalesInvoicePaymentService {
+
+    private readonly salesInvoiceDao;
+    private readonly customerPaymentDao;
+    private readonly salesInvoicePaymentDao;
+
+    constructor() {
+        this.salesInvoiceDao = new SalesInvoiceDao();
+        this.customerPaymentDao = new CustomerPaymentDao();
+        this.salesInvoicePaymentDao = new SalesInvoicePaymentDao();
+    }
+
+    @Get("/salesInvoiceData/:customerPaymentId")
+    public salesInvoiceData(_: any, ctx: any) {
+        const customerPaymentId = ctx.pathParameters.customerPaymentId;
+
+        let customerPayment = this.customerPaymentDao.findById(customerPaymentId);
```

Comment:
Please always use `const` modifier instead of `let` if the value is not going to be assigned to something else later in the code.

Reply:
I understood and fixed that